code/content_packages/corporate/away_sites/lar_maria/lar_maria.dm

```diff
@@ -1,3 +1,4 @@
+#include "../../_corporate.dme"
```

Comment:
I don't recall: does this need the `#ifndef BLAH` gating here to make sure it's not included multiple times?

Reply:
`_corporate.dme` gates it itself.

Reply:
ah.